libraries/engage/components/MessageBar/MessageBar.jsx

```diff
@@ -11,19 +12,20 @@ type DefaultProps = {
 
 type Props = DefaultProps & {
   messages: Message[],
-  classNames?: ClassNames
+  classNames?: ClassNames,
+  raised?: boolean,
 };
 
 /**
  * The MessageBar component.
  * @param {Object} props The component props.
- * @param {Array} props.messages The message content.
- * @param {Object} props.classNames Styling.
+ * @property {Array} props.messages The message content.
+ * @property {Object} props.classNames Styling.
  * @return {JSX}
  */
-const MessageBar = ({ messages, classNames }: Props) => (
+const MessageBar = ({ messages, classNames, raised }: Props) => (
   <div
-    className={css(styles.container, classNames.container)}
+    className={css(raised ? styles.containerRaised : styles.container, classNames.container)}
```

Comment:
When `cartItemsDisplay` is set to `line`, the appearance of the MessageBar changed a little bit.

![message bar](https://user-images.githubusercontent.com/13087792/77333484-c84d1800-6d23-11ea-811d-b71c33695f83.jpg)

The issue is caused by a changed padding of the `container` class from the props.

Compare (before)
https://github.com/shopgate/pwa/blob/46e41d8fe9cd6098f1b7d300b3cb08e15bd52808/themes/theme-ios11/pages/Cart/components/Item/style.js#L8

against the changes at `libraries/engage/cart/components/CartItem/CartItem.style.js` from the PR.

Reply:
Fixed.